Makefile

```diff
@@ -1,5 +1,7 @@
 DIR = ./compile
+SOLC_COMMAND = docker run --rm -i ethereum/solc:0.5.15 solc --standard-json
```

Comment:
How does this even work without the volume mounting?

Reply:
Aaaah, right, I remember, you are actually bundling all the contract code into JSON before. Makes sense. Then this works of course.